Kontent.Ai.Management/IManagementClient.cs

```diff
@@ -689,4 +690,16 @@ public interface IManagementClient
     /// <returns>The <see cref="EnvironmentModel"/> instance that represents the modified environment.</returns>
     Task<EnvironmentModel> ModifyEnvironmentAsync(IEnumerable<EnvironmentOperationBaseModel> changes);
 
+    /// <summary>
+    /// Returns the preview configuration.
+    /// </summary>
+    /// <returns>The <see cref="PreviewConfigurationResponseModel"/> instance that represents the preview configuration.</returns>
+    Task<PreviewConfigurationResponseModel> GetPreviewConfigurationAsync();
+
+    /// <summary>
+    /// Updates the preview configuration.
+    /// </summary>
+    /// <param name="previewConfiguration">Represents configuration that will be used for project.</param>
+    /// <returns>The <see cref="PreviewConfigurationResponseModel"/> instance that represents the preview configuration.</returns>
+    Task<PreviewConfigurationResponseModel> UpdatePreviewConfigurationAsync(PreviewConfigurationRequestModel previewConfiguration);
```

Comment:
Why is it called update here but in docs it's modify? Can we unify it? I am for modify :)

Reply:
Well, it is probably because each thing wrote different person, ok I'll unify it

Reply:
fixed